src/App.js

```diff
@@ -14,6 +15,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Home />}/>
         <Route path="/Articles" element={<Articles />}/>
+        <Route path="/Articles/:article_id" element={<Article />}/>
         <Route path="/Users" element={<Users />}/>
```

Comment:
based on my comment about needing a URL that leads to the topic-filtered list, please have a route here that allows that to happen